src/eth_plugin_handler.c

```diff
@@ -64,7 +66,8 @@ void eth_plugin_prepare_query_contract_UI(ethQueryContractUI_t *queryContractUI,
     }
 
     queryContractUI->screenIndex = screenIndex;
-    strlcpy(queryContractUI->network_ticker, get_tx_network_ticker(), MAX_TICKER_LEN);
+    chain_id = get_tx_chain_id();
+    strlcpy(queryContractUI->network_ticker, get_displayable_ticker(&chain_id), MAX_TICKER_LEN);
```

Comment:
What if we don't find get_displayable_ticker ?

Reply:
`get_displayable_ticker` will always return a non-NULL ticker, in the case that `get_network_ticker_from_chain_id` returns NULL, it will return the current app/clone ticker ("ETH" for Ethereum, "MATIC" for Polygon, ...).